system/nsh/nsh_main.c

```diff
@@ -127,7 +127,8 @@ int main(int argc, FAR char *argv[])
 
   nsh_initialize();
 
-#if defined(CONFIG_NSH_TELNET) && !defined(CONFIG_NETINIT_NETLOCAL)
+#if defined(CONFIG_NSH_TELNET) && !defined(CONFIG_NSH_DISABLE_TELNETSTART) && \
```

Comment:
let's  call nsh_execute to launch system/telnetd in nsh_telnetstart

Reply:
@xiaoxiang781216 
In my understanding, we can not call `nsh_execute` here.

Reply:
Hmm, we need move nsh_telnetstart a little bit later to get vtbl context.